docs/UsersGuide.asciidoc

```diff
@@ -734,13 +738,86 @@ scenarios:
       testsuite: gnome
       machine: [32bit, 64bit-staging]
       settings:
-        FOO: 2
+        FOO: '2'
     opensuse-15.2-GNOME-Live-x86_64:
     - textmode
     - gnome: *gnome
     - gnome_staging: *gnome_staging
 ```
 
+=== YAML Merge Keys
+
+Also link:https://yaml.org/type/merge.html[YAML Merge Keys] are supported.
+This way you can reuse previously defined anchors and add other values to it.
+Values in the merged alias will be overriden.
+
+You can even merge more than one alias.
+
+
+```yaml
+products:
+  opensuse-15.1-DVD-Updates-x86_64:
+    distri: opensuse
+    flavor: DVD-Updates
+    version: '15.1'
+  opensuse-15.2-GNOME-Live-x86_64:
+    distri: opensuse
+    flavor: GNOME-Live
+    version: '15.2'
+scenarios:
+  x86_64:
+    opensuse-15.1-DVD-Updates-x86_64:
+    - textmode
+    - gnome:
+      machine: uefi
+      priorty: 70
+      settings: &common1
+        QEMUVGA: cirrus
+        FOO: default foo
+    - gnome:
+      machine: [32bit, 64bit-staging]
+      priorty: 70
+      settings: &common2
+        QEMUVGA: cirrus
+        FOO: default foo
+        BAR: default bar
+    - gnome_staging:
+      testsuite: gnome
+      machine: [32bit, 64bit-staging]
+      settings:
+        # Merge
+        << : *common1
+        FOO: foo # overrides the value from the merge keys
+    - gnome_staging:
+      testsuite: gnome
+      machine: [32bit, 64bit-staging]
+      settings:
+        # Merge
+        << : [*common1, *common2] # *common1 overrides *common2
+        FOO: foo # overrides the value from the merge keys
+```
+
+
+=== General YAML documentation
+
+The YAML JobTemplates are written in YAML 1.2. In YAML, strings don't have to
```

Comment:
I would use "job templates" instead of "JobTemplates" in documentation. And it is also usually preferred to spell things out ("do not" instead of "don't").